docs/test/replication-controller-list.md

```diff
@@ -0,0 +1,42 @@
+|--------------------------------------|------------------------------|-------------------------------------|-----------------|--------------|-----------------|---------------------------|------------|--------|---------------------------------------------------------------------------------------------|
+| Replication controller count on page | 1                            | odd                                 | even            | 20-30        |                 |                           |            |        |                                                                                             |
+| Screen size                          | normal                       | mobile                              | narrow          | wide         | active resizing |                           |            |        |                                                                                             |
+| Replication controller status        | valid state                  | error state                         |                 |              |                 |                           |            |        | error increases height of the card, error state can be caused by setting non-existing image |
+| Replication controller fields        | normal                       | max length                          | hight pod count |              |                 |                           |            |        |                                                                                             |
+| Replication controller labels        | normal                       | long text (253 prefix and 63 label) | 1               | odd          | even            | short and long text mixed | 20-30      |        | labels increase height of the card                                                          |
+| Actions                              | view details                 | edit pod count                      | delete          | display logs |                 |                           |            |        |                                                                                             |
+| Edit pod count                       | negative                     | 0                                   | 1               | less than 10 | more than 30    | empty                     | scale down | cancel |                                                                                             |
+| Delete replication controller action | normal                       | with checkbox checked               | cancel          |              |                 |                           |            |        |                                                                                             |
```

Comment:
I don't know the algorithm in details, but maybe a deployment is possible (using kubectl) that will produce something like the following:
* the wrong service is deleted
* no service is deleted
* too much is deleted
* error
* nothing happens although the service should be deleted

IMHO, the factor does not encourage deep enough testing. Where should be put this tests? How to describe the factors? 

BTW: The checkbox is also shown if no service exits - I would say this is a bug.